launch-or-update-local.ps1

```diff
@@ -1 +1,25 @@
-helm upgrade bc src/buyingcatalogue -n buyingcatalogue -i -f environments/local-docker.yaml -f local-overrides.yaml
\ No newline at end of file
+####################################################
+### Deploy local cluster                         ###
+### usage:                                       ###
+###  ./launch-or-update-local.ps1                ###
+### or without downloading updates...            ###
+###  ./launch-or-update-local.ps1 -update false  ###
+####################################################
+
+param(
+        [Parameter()]
+        [ValidateSet('','true','false')]
+        [string]$update
```

Comment:
so in shell i can use either -u or -update; but in powershell it's only -update?

Reply:
Good point Tony, I'll add an additional attribute for -u to Powershell also

Reply:
it looks like it's -u and --update on shell. is that right? Can it be the same on powershell?

Reply:
Ah ok, I get what you mean now, I'll sort that